### PR TITLE
Add support and themes for Portugol

### PR DESCRIPTION
**Pull request by marcoaugustoandrade**
Portugol is pseudocode in Portuguese for IDE Portugol Studio.
Portugol Studio Light and Portugol Studio Dark are themes simulation like IDE Portugol Studio theme.

### CODE REVIEW

**Comment by gusbemacbe** [comment]
Unfortunately, Ivan is no longer online since August 2017.

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "syntax"
   ],
   "homepage": "https://highlightjs.org/",
-  "version": "9.12.0",
+  "version": "9.13.0",
```
Comment (ldct) [inline package.json]: please keep the original version (`9.12`)

File: docs/conf.py [hunk 2]
```diff
@@ -48,9 +48,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '9.12'
```
Comment (ldct) [inline docs/conf.py]: please keep the original version (`9.12`)

**Comment by gusbemacbe** [comment]
Remember of making backup of your Portugol file and upgrade your version to the latest version. Please follow the tutorial, Marcos:

1. Open the terminal:
a. `git clone https://github.com/marcoaugustoandrade/highlight.js old-project`
b. `git clone https://github.com/highlightjs/highlight.js new-project`

2. `cd new-project && mkdir new-project && git mv !(new-project) new-project`
3. `git commit -a -S -m “Moving new project into its my own old subdirectory”`

If you see the error, do not worry.

4. `cd ../old-project`
5. `git remote add new-project ../new-project`
6. `git fetch new-project`
7. `git checkout -b feature/merge-new-project`
8. `git merge -S --allow-unrelated-histories new-project/master`

And type your GitHub username and your password.

9. `git push origin feature/merge-new-project`
10. `git remote rm new-project`

11. Open your repository page [here](https://github.com/marcoaugustoandrade/highlight.js)
12. Select the branch and change the branch `master` to `feature/merge-new-project`
13. Click in "pull request", your selected branch (`feature/merge-new-project`) will be selected and compared to.. And select the branch `master` of your repository.
14. Submit the pull request and then merge it to your `master` branch. 
15. Put the backed up file Portugol. 
16. Create the new pull request.

**Comment by marcoscaceres** [comment]
Closing due to inactivity.

**Pull request closed without merging**